docs/features.md

```diff
@@ -2,4 +2,35 @@ Click the below for a walk through of the product functionality:
 
 [![PostHog demo](_media/posthog-demo.png)](https://youtu.be/vGy9gzMAAes-Y?t=35s "PostHog Demo Video")
 
-There is a detailed table of the functionality on the [pricing](/pricing) page.
\ No newline at end of file
+There is a detailed table of the functionality on the [pricing](/pricing) page.
+
+Below is a table of PostHog features and how this compares with the functionality in common alternatives.
+
+
+|  Feature 	|   PostHog	| Amplitude  	|  Mixpanel 	| Heap  	|
+|:-:	|:-:	|:-:	|:-:	|:-:	|
+|   Autocapture	|   ✔	|   	|   	|   ✔	|
+|   API 	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   In-App Events Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Conversion Tracking	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Dashboard	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Data Visualisation	|  ✔ 	|   ✔	|   ✔	|   	|
+|   Cohort Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Funnel Analysis	|   ✔	|   ✔	|   ✔	|   ✔	|
+|   Push Notifications	|   	|   ✔	|   ✔	|   ✔	|
+|   Retention Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Revenue Tracking	|   	|   ✔	|   ✔	|   ✔	|
+|   Uninstall Tracking	|   	|   ✔	|   	|   	|
+|   Data Import	|   	|   ✔	|   ✔	|   	|
+|   Data Export	|   	|   ✔	|   ✔	|   	|
```

Comment:
I'd argue we have both data import and export as we have APIs. What's the difference between this and API's?

Reply:
This is meant from a front-end perspective - i.e. export to csv from front end - I could make this clearer?

From an import perspective - I meant bulk importing existing data (I think this feature is built so that if I want to move to amplitude from mixpanel I can easily move all my historic info across - https://github.com/PostHog/posthog/issues/402)

Reply:
Okay - we do actually have something there so I'll get onto merging that, for now it can stay open